apps/wing-console/console/design-system/src/modal.tsx

```diff
@@ -12,6 +12,25 @@ export interface ModalProps {
   children?: ReactNode;
 }
 
+export const ModalFooter = ({ children }: { children: ReactNode }) => {
```

Comment:
Try to leave the `ModalProps` interface and the `Modal` component together. I'd move ModalFooter and ModalBody before those.